### PR TITLE
Problem: no unit tests for base85 or public curve functions.

### PR DESCRIPTION
**Pull request by evoskuil**

### CODE REVIEW

File: tests/test_base85.cpp [hunk 1]
```diff
@@ -0,0 +1,92 @@
+/*
+    Copyright (c) 2007-2016 Contributors as noted in the AUTHORS file
```
Comment (bluca) [inline tests/test_base85.cpp]: copyright range should be just 2016 since it's a new file

File: tests/test_sodium.cpp [hunk 2]
```diff
@@ -0,0 +1,98 @@
+/*
+    Copyright (c) 2007-2016 Contributors as noted in the AUTHORS file
```
Comment (bluca) [inline tests/test_sodium.cpp]: ditto

**Comment by bluca** [comment]
Thanks! Looks good, but let's get the copyright correct the first time to be safe :-)

File: tests/test_sodium.cpp [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+    const int rc = zmq_curve_public (out_public, secret_key);
+
+#if defined (ZMQ_HAVE_CURVE)
+    assert (rc == 0);
+    assert (zmq_errno () == 0);
+    assert (streq (out_public, public_key));
+#else
+    assert (rc == -1);
+    assert (zmq_errno () == ENOTSUP);
+    (void) public_key
```
Comment (bluca) [inline tests/test_sodium.cpp]: missing ";"

**Comment by bluca** [comment]
If it's not too much trouble, would you mind please squashing the fixup commits into the first one? This helps to keep git bisect working, which is an invaluable tool

**Comment by bluca** [comment]
Thanks! Good to go or do you have more commits coming?

**Comment by evoskuil** [comment]
No more as long as it clears the gate. Thanks for your help.

**Pull request merged**